.github/workflows/autotag.yaml

```diff
@@ -2,16 +2,22 @@ name: Bump version
 on:
   push:
     branches:
-      - main
+      - master
```

Comment:
Are you sure about this one? Shouldn't it stay `main`?

Reply:
```suggestion
      - main
```